hstream/src/HStream/Server/Config.hs

```diff
@@ -189,6 +222,24 @@ parseJSONToOptions CliOptions {..} obj = do
   -- TODO: remove the following 2 options
   let _serverHost     = fromMaybe "0.0.0.0" _serverHost_
   let _ldConfigPath   = _storeConfigPath
+
+  -- TLS config
+  nodeEnableTls <- nodeCfgObj .:? "enable-tls" .!= False
+  nodeTlsKeyPath <- nodeCfgObj .:? "tls-key-path"
+  nodeTlsCertPath <- nodeCfgObj .:? "tls-cert-path"
+  nodeTlsCaPath <- nodeCfgObj .:? "tls-ca-path"
```

Comment:
Can you also add relevant options in `conf/hstream.yaml`

Reply:
Done